CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### [Unreleased][HEAD]
+
+## [v1.0.0-beta.9] - Sep 9th 2019
```

Comment:
You have to construct this link at the bottom of the file by making a named markdown link that compares the two branches: 

```
[v1.0.0-beta.9]: https://github.com/Esri/calcite-components/compare/v1.0.0-beta.8...v1.0.0-beta.9 "v1.0.0-beta.9"
```

Reply:
Oh nifty! Done.